frappe/utils/pdf.py

```diff
@@ -352,6 +353,18 @@ def toggle_visible_pdf(soup):
 		tag.extract()
 
 
+@frappe.whitelist()
+@redis_cache(ttl=60 * 60)
+def is_wkhtmltopdf_valid():
+	try:
+		res = subprocess.check_output(["wkhtmltopdf", "--version"])
+		is_wkhtmltopdf_valid = "qt" in res.decode("utf-8").lower()
+	except Exception:
+		return False
+
+	return is_wkhtmltopdf_valid
+
+
```

Comment:
This can _STILL_ be even shorter and more readable :smile: 

```suggestion
	try:
		output = subprocess.check_output(["wkhtmltopdf", "--version"])
		return "qt" in output.decode("utf-8").lower()
	except Exception:
		return False


```